Tests/LinuxMain.swift

```diff
@@ -0,0 +1,6 @@
+import XCTest
+@testable import UInt128Tests
+
+XCTMain([
+    testCase(UInt128Tests.allTests),
```

Comment:
👎 

UInt128Tests.swift has a lot more test classes than UInt128Tests(). This should include a testCase for each class referencing .allTests for each.